### DISCUSSION TITLE
KAG introduction and applications

### DISCUSSION THREAD

**Discussion by caszkgui** [OpenSPG/KAG#52] [Show and tell]
[知识增强KAG框架与垂域应用.pdf](https://github.com/user-attachments/files/17796529/KAG.pdf)
[Introduction to KAG-en.pdf](https://github.com/user-attachments/files/17843606/Introduction.to.KAG-en-tc-20241111.pdf)

**Comment by mudevai** [comment]
感觉这个才是正确方向，不过单Schema对齐可能还不够用，有没有可能引入RulesEngine呢？

**Reply by leywar** [reply]
RuleEngine相关能力也会陆续在KAG框架中开放，openspg是个知识推理引擎，内置较多规则知识定义和推理能力

**Comment by zlongHe** [comment]
垂类领域还是需要这把利器

**Comment by zhanghanwei2008** [comment]
虽然这方面刚刚接触，研究不是很深，但是感觉有这个框架引入，会大大降低知识图谱与大语言模型相结合开发的门槛，支持~~，加油！！

**Comment by kzaiship** [comment]
为什么mac布置完只能一次上传一个文件呀

**Reply by winer632** [reply]
linux部署完也是一个知识库只能上传一个文件

**Comment by Chelsea-Coder-Fan** [comment]
源码 bug 太多了，建议好好修复下。严重影响使用体验。

**Reply by Chelsea-Coder-Fan** [reply]
server 端有没有计划开源，可以一起修复 bug，开发新功能。

**Reply by patrickleehua** [reply]
一堆bug我也是服了

**Reply by wstars1994** [reply]
唉 真的是

**Reply by a67793581** [reply]
感谢提醒

**Comment by winer632** [comment]
这个项目现在用在蚂蚁的什么业务或者产品上了呢？

**Reply by kakaka89** [reply]
同问

**Comment by tianlinzx** [comment]
kag-model，会开源吗？

**Comment by saisurya98** [comment]
thank you for pdf.
